.github/workflows/ci.yml

```diff
@@ -4,11 +4,14 @@ on:
   pull_request:
   push:
     branches:
-      - master
+    - master
+  schedule:
+    - cron: "0 0 1 */1 *"
 
 jobs:
   test:
-    runs-on: ubuntu-16.04
+    name: Elixir ${{matrix.pair.elixir}} / OTP ${{matrix.pair.otp}}
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-20.04
```

Reply:
Using Ubuntu `latest` is actually the main purpose of this PR 😅 

This repository seems to have very rare change frequency, which explains why it was sitting on removed image version for 6 months (so CI was defunct for this time period). By not pinning to a particular version, this situation highly probably won't happen in the future.

Also, there are no hard dependencies on Ubuntu version "specific" things in this CI pipeline, as it's just interfacing with Mix. I expect any Ubuntu change handling will be solved by Elixir setup action internally, so I don't see any benefits of pinning to a particular Ubuntu version.

Though I understand if you have any strict conventions in this area.